apps/server/src/modules/oauth/uc/oauth.uc.spec.ts

```diff
@@ -341,7 +346,7 @@ describe('OAuthUc', () => {
 					});
 					userMigrationService.migrateUser.mockResolvedValue(userMigrationDto);
 
-					await expect(uc.migrate('currentUserId', query, system.id as string)).rejects.toThrow(error);
+					await expect(uc.migrate('jwt', 'currentUserId', query, system.id as string)).rejects.toThrow(error);
 				});
```

Comment:
If this.cacheManager.del rejects?

Reply:
The cache manager is no concern for the uc, since the uc does not know that the cache manager is even called. Even if the authenticationService.removeJwtFromWhitelist method rejects, there is nothing we can do to recover from this point and the uc method itself rejects. This is the default behavior of JavaScript and needs no testing.

Reply:
"here is nothing we can do to recover from this point and the uc method itself rejects" so this the desired behavior and this should be tested in a case like "returns error".

Reply:
Then I would have to write a test for every `await` that I do, because they can all reject. This is not done anywhere in the project. I think it is a bit over the top, but if you insist on it, then we can talk about it in person.